services/storage-users/pkg/command/uploads.go

```diff
@@ -311,3 +324,181 @@ func buildInfo(filter storage.UploadSessionFilter) string {
 	b.WriteString(":")
 	return b.String()
 }
+
+// DeleteStaleProcessingNodes is the entry point for the delete-stale-nodes command
+func DeleteStaleProcessingNodes(cfg *config.Config) *cli.Command {
+	return &cli.Command{
+		Name:  "delete-stale-nodes",
+		Usage: "Delete all nodes in processing state that are not referenced by any upload session",
+		Flags: []cli.Flag{
+			&cli.StringFlag{
+				Name:     "spaceid",
+				Usage:    "Space ID to check for processing nodes (omit to check all spaces)",
+				Required: false,
+			},
+			&cli.BoolFlag{
+				Name:  "dry-run",
+				Usage: "Only show what would be deleted without actually deleting",
+				Value: false,
+			},
+			&cli.BoolFlag{
+				Name:  "verbose",
+				Usage: "Enable verbose logging",
+				Value: false,
+			},
+		},
+		Before: func(c *cli.Context) error {
+			return configlog.ReturnFatal(parser.ParseConfig(cfg))
+		},
+		Action: func(ctx *cli.Context) error {
+			start := time.Now()
+			spaceIDs := []string{}
+
+			// Check if specific space ID provided
+			if ctx.IsSet("spaceid") {
+				spaceID := ctx.String("spaceid")
+				spaceIDs = append(spaceIDs, spaceID)
```

Comment:
```suggestion
				spaceIDs = append(spaceIDs, ctx.String("spaceid"))
```
Using one line makes it look more goish 😉